packages/datadog_flutter_plugin/lib/datadog_flutter_plugin_web.dart

```diff
@@ -52,19 +52,40 @@ class DatadogSdkWeb extends DatadogSdkPlatform {
   Future<void> addUserExtraInfo(Map<String, Object?> extraInfo) async {}
 
   @override
-  Future<void> initialize(
+  Future<PlatformInitializationResult> initialize(
     DdSdkConfiguration configuration, {
     LogCallback? logCallback,
     required InternalLogger internalLogger,
   }) async {
-    if (configuration.loggingConfiguration != null) {
-      DdLogsWeb.initLogs(configuration);
+    bool logsInitialized = false;
+    try {
+      if (configuration.loggingConfiguration != null) {
+        DdLogsWeb.initLogs(configuration);
+        logsInitialized = true;
+      }
+    } catch (e) {
+      internalLogger.warn('DatadogSdk failed to initialize logging: $e');
+      internalLogger.warn(
+          'Did you remember to add "datadog-logs-v4.js" to your scripts?');
```

Comment:
> datadog-logs-v4.js

Might as well not reference a specific file/version, it will make the message rot sooner.